lib/configuration.js

```diff
@@ -119,6 +121,14 @@ Configuration.prototype.load = function (filePath, callback) {
           return callback(error)
         }
 
+        config.conf.passwordPolicy = Object.assign({
+          length: 8,
+          digits: 1,
+          lowercaseLetters: 1,
+          uppercaseLetters: 1,
+          nonAlphaNumeric: 0
+        }, config.conf.passwordPolicy);
```

Comment:
If these are defaults going forward, I think there needs to be an overall config you can set to override for dev/test installs:
```js
usePasswordPolicy: false
```

Reply:
Not sure about that. You could simply set every value to 0 to have the same effect.